values/prometheus-operator/rules/harbor-db-backup.yaml

```diff
@@ -0,0 +1,10 @@
+groups:
+  - name: harbor-db-backup
+    rules:
+      - alert: BackupHarborDbFailing
+        expr: time() - cnpg_collector_last_failed_backup_timestamp{namespace="harbor",pod=~"harbor-otomi-db-1"} > 86400 or time() - cnpg_collector_last_failed_backup_timestamp{namespace="harbor",pod=~"harbor-otomi-db-2"} > 86400
```

Comment:
It should be `or` because when backup is configured only one will actually be backed up (or not). So in the above the backup failed for `harbor-otomi-db-2` while one (harbor-otomi-db-1) will not give an error. In this case the backup failed. But we don't know if the backup is done for 1 or 2. That's why the `or`.

Reply:
I have backups for harbor completed every hour. Still, the alter is firing.
![image](https://github.com/user-attachments/assets/23f92f42-3063-4f9d-8291-6e462f5ca91b)

Reply:
I see it working.

Reply:
After deploying it on a fresh cluster I see that the alerts works correctly.